packages/react-notifications/src/NotificationsModal.tsx

```diff
@@ -2,11 +2,14 @@ import { Modal } from '@ttoss/components';
 import { NotificationsBox } from './NotificationsBox';
 import { useNotifications } from './Provider';
 
-export const NotificationsModal = () => {
+export const NotificationsModal = ({ zIndex }: { zIndex?: number }) => {
```

Comment:
Instead of exposing only `zindex`, why not expose the entire Modal style?

Reply:
I think we can make it by now. The idea here is to not expose every detail implementation for the user, only what's really necessary to not conflict with other UI element.

Reply:
I think we can advance without this exposition.
Until there, some UI's will be slightly buggy, as demonstrated on the picture on #370

Reply:
No problem! A buggy UI makes the design prioritize this issue. Would you mind holding this PR a little bit?